protostar/cheatable_starknet/callable_hint_locals/start_prank_hint_local.py

```diff
@@ -13,13 +13,13 @@ def __init__(self, contracts_controller: ContractsController):
 
     @property
     def name(self) -> str:
-        return "prank"
+        return "start_prank"
 
     def _build(self) -> Callable:
         return self.prank
 
-    def prank(self, caller_address: int, target_address: int):
-        valid_target_address = Address.from_user_input(target_address)
+    def prank(self, caller_address: int, target_contract_address: int):
```

Comment:
so `prank` or `start_prank`, please commit and use one consistently 🥺

Reply:
`start_prank` vs `prank` semantics are from foundry cheatcodes, if we use the names we should stick to what they do in foundry as well.

Reply:
It is leftover from the previous code. I will change it to `start_prank` here.